frontend/src/nc/pages/components/AddModal.js

```diff
@@ -210,7 +219,7 @@ const AddModal = ({ setModal, table, columns_, postEndpoint, refreshFunc }) => {
             &times;
           </span>
         </div>
-        <form className="flex flex-col mb-4" onSubmit={handleSubmit}>
+        <form className="flex flex-col mb-4 overflow-y-auto overflow-x-hidden max-h-[500px]" onSubmit={handleSubmit}>
```

Comment:
can you use this kind of an overflow class in place of in-line styling in Dashboard.jsx?